requirements.txt

```diff
@@ -15,3 +16,4 @@ datasets
 tiktoken
 wandb
 tqdm
+google-cloud-aiplatform
```

Comment:
I'm thinking this might be heavy weight for all general users, shall we move these additional reqs to just the vertex section? I will do the same for bedrock.

Reply:
do you mean to take it out of requirements.txt and only add it as an information in the README?

Reply:
Yeah, let's leave requirements as clean `anthropic` and put the additional reqs in the README.

Reply:
sounds good